filebeat/docs/modules/fortinet.asciidoc

```diff
@@ -37,20 +37,26 @@ include::../include/config-option-intro.asciidoc[]
 ----
 - module: fortinet
   firewall:
-    var.syslog_host: 0.0.0.0
-    var.syslog_port: 9004
+    enabled: true
+    var.input: udp
+    var.host: 0.0.0.0
+    var.port: 9004
 ----
 
 include::../include/var-paths.asciidoc[]
 
-*`var.syslog_host`*::
+*`var.input`*::
 
-The interface to listen to UDP based syslog traffic. Defaults to localhost.
+The protocol to use, can be either the value `tcp` or `udp`.
```

Comment:
What about mentioning `file`?

Reply:
> Did you manually check the udp input with a netcat message?

I tested with both TCP and UDP. TCP works with the same amount of messages and 0 error.messages filled, UDP also works but I see a few GROK missmatch messages (with the same logs). This seems to be because it sends the whole 300 messages at the same time, and all messages that gets through gets parsed correctly, while the other few gets a grok error message.

Any idea here? Testing seems fine, and If I tested with a smaller set of logs it should still work fine.

Reply:
Did some new testing, UDP is also fine.